app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/ItemListingScreen.kt

```diff
@@ -453,6 +459,8 @@ private fun ItemListingContent(
                             onItemClick = { onItemClick(it.authCode) },
                             onEditItemClick = { onEditItemClick(it.id) },
                             onDeleteItemClick = { onDeleteItemClick(it.id) },
+                            onMoveToBitwardenClick = { onMoveToBitwardenClick(it.id) },
```

Comment:
Can we get a test for this click event?

Reply:
Heh, I can't figure out why its saying these lines aren't covered, because I did in fact write a test for this click event:

https://github.com/bitwarden/authenticator-android/pull/250/files#diff-7722ba9e365f1de663dc3a28e311a791a46a642ad620c03b258f935d5e7caea7R203-R222

Reply:
Must have been something stale, looks good now!